### PR TITLE
Update hfss3dlayout.py

### PR DESCRIPTION
**Pull request by PipKat**
Add missing parameters and edit back ticks.

### CODE REVIEW

File: pyaedt/hfss3dlayout.py [hunk 1]
```diff
@@ -219,22 +240,23 @@ def create_pin_port(self,name,xpos=0, ypos=0, rotation=0, top_layer=None, bot_la
...
             Rotation of the pin in degrees. The default is ``0``.
-        top_layer : str
-            Top layer of the pin. If ``None``, it is automatically assigned to the top. The default is ``None``.
+        top_layer : str, optional
+            Top layer of the pin. The default is ``None``, in which case the top 
+            layer is assigned automatically.
         bot_layer : str
-            Bottom layer of the pin. If ``None``, it is automatically assigned to the bottom. The default is ``None``.
+            Bottom layer of the pin. The default is ``None``, in which case the
+            botton is assigned automatically.
```
Comment (PipKat) [inline pyaedt/hfss3dlayout.py:254]:
```suggestion
            bottom is assigned automatically.
```

File: pyaedt/hfss3dlayout.py [hunk 2]
```diff
@@ -11,18 +11,25 @@
...
-    """Generate a sweep string like this example: ``"LIN 10GHz 20GHz 0.05GHz LINC 20GHz 30GHz 10 DEC 30GHz 40GHz 10 40GHz``"""
+    """SweepString class.
+    
+    This class allows you to generate a sweep string like in this example: 
+    ``"LIN 10GHz 20GHz 0.05GHz LINC 20GHz 30GHz 10 DEC 30GHz 40GHz 10 40GHz``
+    
+    Parameters
+    ----------
+    unit : str, optional
+        Units for the sweep string. Thedefault is ``"GHz"``.
```
Comment (akaszynski) [inline pyaedt/hfss3dlayout.py:22]:
```suggestion
        Units for the sweep string. The default is ``"GHz"``.
```

File: pyaedt/hfss3dlayout.py [hunk 3]
```diff
@@ -78,45 +85,60 @@ class Hfss3dLayout(FieldAnalysis3DLayout):
...
+        the active version or latest installed version is used.
+    NG : bool, optional
+        Whether to run AEDT in the non-graphical mode. The default 
+        is``False``, which launches AEDT in the graphical mode.  
+    AlwaysNew : bool, optional
+        Whether to launch an instance of AEDT in a new thread, even if
+        another instance of the ``specified_version`` is active on the
+        machine.  The default is ``True``.
+    release_on_exit : bool, optional
+        Whether to release  AEDT on exit.
```
Comment (akaszynski) [inline pyaedt/hfss3dlayout.py:115]:
```suggestion
        Whether to release AEDT on exit. 
```

File: pyaedt/hfss3dlayout.py [hunk 4]
```diff
@@ -143,9 +165,9 @@ def create_edge_port(self, primivitivename, edgenumber, iscircuit=True):
         primivitivename : str
             Name of the primitive.
         edgenumber :
-            Edge number on which to create the port.
-        iscircuit : bool
-            Indicates if it is a circuit port. The default is ``False``.
+            Edge number to create the port on.
+        iscircuit : bool, optional
+            Whethher the edge port is a circuit port. The default is ``False``.
```
Comment (akaszynski) [inline pyaedt/hfss3dlayout.py:170]:
```suggestion
            Whether the edge port is a circuit port. The default is ``False``.
```

File: pyaedt/hfss3dlayout.py [hunk 5]
```diff
@@ -171,29 +193,28 @@ def create_coax_port(self, vianame, layer, xstart, xend,ystart, yend, archeight=
...
+            Starting position of the pin on the X axis.
         xend :
-            X-axis position of the pin.
+            Ending position of the pin on the X axis.
+        ystart :
+            Starting position of the pin on the Y axis.
         yend :
-            Y-axis end position of the pin.
+            Ending postiion of the pin on the Y axis.
         archeight :
```
Comment (akaszynski) [inline pyaedt/hfss3dlayout.py:207]:
```suggestion
        archeight : float, optional
```

File: pyaedt/hfss3dlayout.py [hunk 6]
```diff
@@ -171,29 +193,28 @@ def create_coax_port(self, vianame, layer, xstart, xend,ystart, yend, archeight=
...
             Arc height. The default is ``0``.
         arcrad :
```
Comment (akaszynski) [inline pyaedt/hfss3dlayout.py:209]:
```suggestion
        arcrad : float, optional
```

File: pyaedt/hfss3dlayout.py [hunk 7]
```diff
@@ -171,29 +193,28 @@ def create_coax_port(self, vianame, layer, xstart, xend,ystart, yend, archeight=
...
-            Rotation of the pin in rad. The default is ``0``.
+            Rotation of the pin in radians. The default is ``0``.
         isexternal : bool
```
Comment (akaszynski) [inline pyaedt/hfss3dlayout.py:211]:
```suggestion
        isexternal : bool, optional
```

File: pyaedt/hfss3dlayout.py [hunk 8]
```diff
@@ -219,22 +240,23 @@ def create_pin_port(self,name,xpos=0, ypos=0, rotation=0, top_layer=None, bot_la
...
         name : str
             Name of the pin port.
-        xpos :
+        xpos : float, optional
             X-axis position of the pin. The default is ``0``.
-        ypos :
+        ypos : float, optional
             Y-axis position of the pin. The default is ``0``.
-        rotation :
+        rotation : float
```
Comment (akaszynski) [inline pyaedt/hfss3dlayout.py:247]:
```suggestion
        rotation : float, optional
```

File: pyaedt/hfss3dlayout.py [hunk 9]
```diff
@@ -302,17 +326,17 @@ def validate_full_design(self, name=None, outputdir=None, ports=None):
...
+            Output directory to save the log file to. The default is ``None``.
+        ports : str, optional
+            Number of excitations that are expected. The default is ``None``.
 
         Returns
         -------
-        type
-            All the info in a list for later use
+        list
+            List of all information for later use.
```
Comment (akaszynski) [inline pyaedt/hfss3dlayout.py:339]:
```suggestion
            List of validation message.
```

File: pyaedt/hfss3dlayout.py [hunk 10]
```diff
@@ -548,33 +573,32 @@ def create_frequency_sweep(self, setupname, unit, freqstart, freqstop, num_of_fr
...
-        save_rad_fields_only : bool
-            Save only the radiated fields if ``save_fields = True``. The default is ``False``.
-        use_q3d_for_dc : bool
-            Use Q3D to solve the DC point. The default is ``False``.
+        interpolation_max_solutions : int, optional
+            Maximum number of solutions evaluated for the interpolation process. The default is ``250``.
+        save_fields : bool, optional
+            Whether to save the fields for a discrete sweep only. The default is ``True``.
+        save_rad_fields_only : bool, optional
+            Whether to save only the radiated fields if ``save_fields = True``. The default is ``False``.
```
Comment (akaszynski) [inline pyaedt/hfss3dlayout.py:594]:
```suggestion
            Whether to save only the radiated fields if ``save_fields=True``. The default is ``False``.
```

No spaces for input parameters.

**Review by akaszynski — APPROVED** [review]
Approved with suggestions.

File: pyaedt/hfss3dlayout.py (hunk 9 above)
Comment (PipKat) [inline pyaedt/hfss3dlayout.py:339]:
```suggestion
            List of validation messages.
```

**Pull request merged**